### PR TITLE
Handle dropdowns default value

### PR DESCRIPTION
**Pull request by Aeto-J**
Hello,
-Enhancement of single dropdown default value
-add "getDropdownName" in function "showSummary" for default value on single dropdown

Thank you,
regards Aeto-J

### CODE REVIEW

File: ajax/dropdownDefaultValue.php [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+}
+
+$table = getTableForItemType($itemtype);
+$rand = $_POST["rand"] ?? mt_rand();
+// Message for post-only
+if (!isset($_POST["admin"]) || ($_POST["admin"] == 0)) {
+    echo "<span class='text-muted'>" .
+    __('Enter the first letters (user, item name, serial or asset number)')
+    . "</span>";
+}
```
Comment (cedric-anne) [inline ajax/dropdownDefaultValue.php:58]: Search field may be not display if elements count is lower than `$CFG_GLPI['ajax_limit_count']`. Also, this dropdown acts the same as other GLPI dropdowns, so adding a hint for this here does not seems really usefull, as fields would probably be configured by admins that are aware of how it should work.

**Review by cedric-anne — APPROVED** [review]
I simplified the code by calling the ajax endpoint in both creation and edition case. I moved many code into this ajax endpoint so there is no need to do display/hide operations using JS code (except for labels).
I also added support of default value for "custom dropdown" type.

**Comment by cedric-anne** [comment]
@Aeto-J @tomolimo 

I am waiting for your confirmation that everything is OK to merge this PR.

**Comment by Aeto-J** [comment]
For me it's good

**Pull request merged**